angular.md

```diff
@@ -0,0 +1,37 @@
+# Angular CLI
+
```

Comment:
Could you please raise a single cheat sheet at a time? That makes easy to review

Reply:
ok